### PR TITLE
[OSDEV-862] Create anonymizer script in docker

### PR DESCRIPTION
**Pull request by VitalyyP**
Create anonymize script in scripts, create anonymizer with scripts and docker-compose.yml

### CODE REVIEW

File: anonymizer/scripts/anonymize [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+# Drop anondb database
+docker exec -i anonymizer-database-1 createdb -U anondb anondb
+
+# Restore anondb database from the restore file
+# -- docker exec -i anonymize_database-1 pg_restore -U anondb -d anondb < "$RESET_FILE_PATH"
+docker exec -i anonymizer-database-1 pg_restore -U anondb -d anondb < ../db_before/dump.dump
+
+# SQL script for updating email fields
+SQL_SCRIPT="DO \$\$
```
Comment (killev) [inline anonymizer/scripts/anonymize:25]: It would be great to copy this script into docker image.

File: anonymizer/scripts/anonymize [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                    phone_number = CASE WHEN email NOT LIKE ''%@speedandfuncion.com'' AND email NOT LIKE ''%@opensupplyhub.com'' THEN md5(random()::text) ELSE phone_number END';
+            END IF;
+        END IF;
+    END LOOP;
+END \$\$;"
+
+
+# Execute SQL script and create a dump file
+# -- docker exec -it anonymize_database-1 psql -U anondb -d anondb -c "$SQL_SCRIPT" && sudo docker exec -it anonymize_database-1 pg_dump -U anondb -d anondb -f "$DUMP_FILE_PATH"
+docker exec -it anonymize_database-1 psql -U anondb -d anondb -c "$SQL_SCRIPT" && sudo docker exec -it anonymize_database-1 pg_dump -U anondb -d anondb -f ../db_after/anonymized_dump.dump
```
Comment (killev) [inline anonymizer/scripts/anonymize:65]: Copy script that anonymise database to the Docker image

File: scripts/anonymize [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cd anonymizer || exit
```
Comment (killev) [inline scripts/anonymize:3]: This is weird, I don't understand why is't needed

File: anonymizer/scripts/run_commands.sh [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+
+./scripts/anonymize
```
Comment (killev) [inline anonymizer/scripts/run_commands.sh:4]: Why we need this? I believe everything should be run inside the container/image

**Review by killev — CHANGES_REQUESTED** [review]
-1
Name of PR is inaccurate.
I few comments regarding implementation added

**Review by killev — DISMISSED** [review]

**Review by vladsha-dev — CHANGES_REQUESTED** [review]
1. Please, update the branch to the top of the parent branch.
2. Please complete the release notes file.

**Review by vladsha-dev — APPROVED** [review]

**Review by killev — APPROVED** [review]

**Pull request merged**